modules/apps/collaboration/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp

```diff
@@ -177,7 +177,14 @@ if (Validator.isNotNull(keywords)) {
 		</liferay-util:buffer>
 
 		<div class="drop-enabled drop-zone no-border">
-			<strong><liferay-ui:message arguments="<%= selectFileHTML %>" key="drag-and-drop-to-upload-or-x" /></strong>
+			<c:choose>
+				<c:when test="<%= BrowserSnifferUtil.isMobile(request) %>">
+					<%= selectFileHTML %>
+				</c:when>
+				<c:otherwise>
+					<strong><liferay-ui:message arguments="<%= selectFileHTML %>" key="drag-and-drop-to-upload-or-x" /></strong>
```

Comment:
Should we use CSS to control this text being bold or should this be wrapped in ```<strong />``` tags?